stable/nginx-ingress/templates/controller-service.yaml

```diff
@@ -57,3 +57,9 @@ spec:
     component: "{{ .Values.controller.name }}"
     release: {{ .Release.Name }}
   type: "{{ .Values.controller.service.type }}"
+{{- if .Values.controller.service.externalTrafficPolicy }}
```

Comment:
Make this `{{- if and (ge .Capabilities.KubeVersion.Minor "7") (.Values.controller.service.externalTrafficPolicy) }}` so v1.6 (and lower) is still supported.